api/tests/integration/application/campaigns/index_test.js

```diff
@@ -58,19 +58,31 @@ describe('Integration | Application | Route | campaignRouter', () => {
 
   describe('GET /api/campaigns/{id}', () => {
 
-    it('should exist', function() {
+    it('should return a 200', function() {
       // when
       const promise = server.inject({
         method: 'GET',
-        url: '/api/campaigns/FAKE_ID',
+        url: '/api/campaigns/1',
       });
 
       // then
       return promise.then((res) => {
-        expect(res.statusCode).to.equal(201);
+        expect(res.statusCode).to.equal(200);
       });
     });
 
+    it('should return a 400 - Bad Request', function() {
```

Comment:
Je rajouterais dans le `it` : "should return a 400 - Bad Request **when campaign code is invalid**"